privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/pixels/PrivacyDashboardRemoteFeature.kt

```diff
@@ -27,7 +27,4 @@ import com.duckduckgo.feature.toggles.api.Toggle
 interface PrivacyDashboardRemoteFeature {
```

Comment:
Is this feature even needed anymore? 

cc/ @aitorvs what happens when we remove features from the code, I take the shared prefs, etc stay, should we delete?

Reply:
@marcosholgado - yeah, happy to remove this entirely.